CellProfiler64.iss

```diff
@@ -45,7 +45,6 @@ Name: "{userappdata}\CellProfiler\ijplugins"; Flags: uninsneveruninstall
 Source: ".\dist\CellProfiler.exe"; DestDir: "{app}"; Flags: ignoreversion
 Source: ".\dist\analysis_worker.exe"; DestDir: "{app}"; Flags: ignoreversion
```

Comment:
Is this still needed? Couldn’t we mimic the Linux and OS X behavior and run `python CellProfiler.py`?